### ISSUE TITLE
Regex parse error

### ISSUE THREAD

**Issue by zolomatok** [DanielXMoore/Civet#337]
The following runs into a parse error towards the closing `]`.
Works in CofeeScript though.
```
let str = 'blah'
if /^\/files\/[a-zA-Z0-9-/.]+$/.test str
  consol.log 'ya'
```

Civet **0.5.64**

**Comment by STRd6** [comment]
Good catch, thanks for the report!

Fixed in v0.5.65

**Comment by zolomatok** [comment]
Mah pleasure! Lightning quick fix as always, I really appreciate it!
